### PR TITLE
[FIX] repair: Respect account.group_show_line_subtotals_tax_included

### PR DESCRIPTION
**Pull request by hubvd**
Steps to follow

- Enable group_show_line_subtotals_tax_included
- Create a repair order and add a tax to a line
-> The subtotal doesn't contain the tax amount

opw-2513287

### CODE REVIEW

File: addons/repair/models/repair.py [hunk 1]
```diff
@@ -581,7 +581,8 @@ class RepairLine(models.Model):
     product_id = fields.Many2one('product.product', 'Product', required=True)
     invoiced = fields.Boolean('Invoiced', copy=False, readonly=True)
     price_unit = fields.Float('Unit Price', required=True, digits='Product Price')
-    price_subtotal = fields.Float('Subtotal', compute='_compute_price_subtotal', store=True, digits=0)
+    price_subtotal = fields.Float('Subtotal', compute='_compute_price_total_and_subtotal', store=True, digits=0)
+    price_total = fields.Float('Total', compute='_compute_price_total_and_subtotal', store=False, compute_sudo=True, digits=0)
```
Comment (Whenrow) [inline addons/repair/models/repair.py:585]: computed fields are non store by default so you can skip this parameter. Same for compute sudo

File: addons/repair/models/repair.py [hunk 2]
```diff
@@ -617,11 +618,12 @@ def constrain_lot_id(self):
         for line in self.filtered(lambda x: x.product_id.tracking != 'none' and not x.lot_id):
             raise ValidationError(_("Serial number is required for operation line with product '%s'") % (line.product_id.name))
 
-    @api.depends('price_unit', 'repair_id', 'product_uom_qty', 'product_id', 'repair_id.invoice_method')
-    def _compute_price_subtotal(self):
+    @api.depends('price_unit', 'repair_id', 'product_uom_qty', 'product_id', 'tax_id', 'repair_id.invoice_method')
+    def _compute_price_total_and_subtotal(self):
```
Comment (Whenrow) [inline addons/repair/models/repair.py:622]: As the PR is in stable you should leave the method signature unchanged. Maybe someone has overridden it

File: addons/repair/models/repair.py (hunk 1 above)
Comment (hubvd) [reply addons/repair/models/repair.py]: compute_sudo is False by default for non stored fields, but I removed the store parameter.

File: addons/repair/models/repair.py (hunk 2 above)
Comment (hubvd) [reply addons/repair/models/repair.py]: Done

**Comment by Whenrow** [comment]
@odoo/upgrade Hello, We cannot add computed field in stable anymore or is it a side effect of a new check ?

**Comment by KangOl** [comment]
> @odoo/upgrade Hello, We cannot add computed field in stable anymore or is it a side effect of a new check ?

You should ask @odoo/rd-runbot to add an exception. It's because the runbot will try to test the upgrade of your branch against the next version, in which your code is not forward-ported yet (as it should actually try to upgrade the previous version to your branch).

**Comment by Xavier-Do** [comment]
> (as it should actually try to upgrade the previous version to your branch).

Actually it does both ;)

**Comment by Whenrow** [comment]
@KangOl Thanks
@Xavier-Do Can we have the exception ?

**Comment by Xavier-Do** [comment]
> @KangOl Thanks
> @Xavier-Do Can we have the exception ?

The decision is not mine, we are discussing about that on discord ;)

**Pull request closed without merging**